### PR TITLE
added problems

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: Lab 5 - Shart Operatorlari.md [hunk 1]
```diff
@@ -891,3 +891,216 @@ int main()
   
 ---
   
+  ### Masala 22
+
```
Comment (wahid-d) [inline Lab 5 - Shart Operatorlari.md:895]: ortiqcha bosh qator

File: Lab 5 - Shart Operatorlari.md [hunk 2]
```diff
@@ -891,3 +891,216 @@ int main()
...
+
+  ##### You Tube
+
+  ##### Description
+
+ >Wahid amaki yaqinda Janubiy Koreadan qaytdi. U har kuni You Tubedan video ko'rishni yaxshi ko'radi. Lekin internet tezligi O'zbekistonda juda past. Videoning davomiyligi T sekund, videoning hajmi har sekund uchun S ga teng, Wahid amakining internet tezligi sekundiga C ga teng. Internet tezligi har doim video hajmidan kichkina `(C < S)`. Wahid amaki videoni uzulishlarsiz ko'rmoqchi. Wahid amaki videoni uzulishlarsiz ko'rishi uchun kamida necha sekund kutib turish kerak.
```
Comment (wahid-d) [inline Lab 5 - Shart Operatorlari.md:901]:
```suggestion
 > Wahid amaki yaqinda Janubiy Koreadan qaytdi. U har kuni You Tubedan video ko'rishni yaxshi ko'radi. Lekin internet tezligi O'zbekistonda juda past. Videoning davomiyligi T sekund, videoning hajmi har sekund uchun S ga teng, Wahid amakining internet tezligi sekundiga C ga teng. Internet tezligi har doim video hajmidan kichkina `(C < S)`. Wahid amaki videoni uzulishlarsiz ko'rmoqchi. Wahid amaki videoni uzulishlarsiz ko'rishi uchun kamida necha sekund kutib turish kerak.
```

File: Lab 5 - Shart Operatorlari.md [hunk 3]
```diff
@@ -891,3 +891,216 @@ int main()
...
+
+ ###### Input
+
```
Comment (wahid-d) [inline Lab 5 - Shart Operatorlari.md:904]: ortiqcha

File: Lab 5 - Shart Operatorlari.md [hunk 4]
```diff
@@ -891,3 +891,216 @@ int main()
...
+
+ Bir qatorda uchta son `S`, `C`, `T`.`(1 < S, C, T < 1000, S > C)`
```
Comment (wahid-d) [inline Lab 5 - Shart Operatorlari.md:906]:
```suggestion
 Bir qatorda uchta sonlar `S`, `C`, `T`: 
 $$1 \le S, C, T \le 1000$$
 $$S \ge C$$
```

File: Lab 5 - Shart Operatorlari.md [hunk 5]
```diff
@@ -891,3 +891,216 @@ int main()
...
+
+ ###### Output
+
+
```
Comment (wahid-d) [inline Lab 5 - Shart Operatorlari.md:910]: ortiqcha

File: Lab 5 - Shart Operatorlari.md [hunk 6]
```diff
@@ -891,3 +891,216 @@ int main()
...
+ Wahid amaki kutishi kerak bo'lgan eng kam vaqt`(butun son sekundlarda)`.
```
Comment (wahid-d) [inline Lab 5 - Shart Operatorlari.md:911]:
```suggestion
 Wahid amaki kutishi kerak bo'lgan eng kam vaqt `(butun son sekundlarda)`.
```

File: Lab 5 - Shart Operatorlari.md [hunk 7]
```diff
@@ -891,3 +891,216 @@ int main()
...
+
+ | Input | Output |
+ | - | - |
+ | `67` `66` `67` | `2` |
+
+ ---
+
+ ### Masala 23
+
+
```
Comment (wahid-d) [inline Lab 5 - Shart Operatorlari.md:929]: ortiqcha

File: Lab 5 - Shart Operatorlari.md [hunk 8]
```diff
@@ -891,3 +891,216 @@ int main()
...
+ ##### Uychalar
```
Comment (wahid-d) [inline Lab 5 - Shart Operatorlari.md:930]:
```suggestion
 ### Masala 23 - Uychalar
```

File: Lab 5 - Shart Operatorlari.md [hunk 9]
```diff
@@ -891,3 +891,216 @@ int main()
...
+
+ ##### Description
+
+ >Ko'p bolalar yoshliglarida gugurt cho'plardan foydalanib turli hil narsalar yasashgan. Masalan, Wahid amaki gugurt cho'plaridan uychalar yasashni yoqtirar edilar. Ularning yasagan uychalarini ko'rinishi quyidagiga o'xshar edi. 1 ta uycha yasash uchun 6 ta cho'p, 2 ta uycha uchun 11 ta cho'p, 3 ta uycha uchun 16 ta cho'p kerak bo'lar edi.
```
Comment (wahid-d) [inline Lab 5 - Shart Operatorlari.md:934]:
```suggestion
 > Ko'p bolalar yoshliglarida gugurt cho'plardan foydalanib turli hil narsalar yasashgan. Masalan, Wahid amaki gugurt cho'plaridan uychalar yasashni yoqtirar edilar. Ularning yasagan uychalarini ko'rinishi quyidagiga o'xshar edi. 1 ta uycha yasash uchun 6 ta cho'p, 2 ta uycha uchun 11 ta cho'p, 3 ta uycha uchun 16 ta cho'p kerak bo'lar edi.
```

File: Lab 5 - Shart Operatorlari.md [hunk 10]
```diff
@@ -891,3 +891,216 @@ int main()
...
+
+ >Kiritilgan**N**ta uycha kerak bo'ladigan cho'plar sonini aniqlaydigan dastur tuzing.
```
Comment (wahid-d) [inline Lab 5 - Shart Operatorlari.md:936]:
```suggestion
 > Kiritilgan**N**ta uycha kerak bo'ladigan cho'plar sonini aniqlaydigan dastur tuzing.
```

File: Lab 5 - Shart Operatorlari.md [hunk 11]
```diff
@@ -891,3 +891,216 @@ int main()
...
+ ![Alt Text](https://oj.ilmhub.uz/public/upload/6f3dc01ed2.png)
+
+ ###### Input
+
+ $0\leq N \leq10^{12}$
+
+ ###### Output
+
+ **N**ta uycha uchun kerak bo'ladigan cho'plar soni
+
```
Comment (wahid-d) [inline Lab 5 - Shart Operatorlari.md:947]: ortiqcha

File: Lab 5 - Shart Operatorlari.md [hunk 12]
```diff
@@ -891,3 +891,216 @@ int main()
...
+ | Input | Output |
+ | - | - |
+ | `5` | `26` |
+
+ ---
+
+ ### Masala 24
+
+ ##### Prize
```
Comment (wahid-d) [inline Lab 5 - Shart Operatorlari.md:965]:
```suggestion
 ### Masala 24 - Prize
```

File: Lab 5 - Shart Operatorlari.md [hunk 13]
```diff
@@ -891,3 +891,216 @@ int main()
...
+
+ ##### Description
+
+ >Shahboz, Muhammadrizo, Muhammadjon Furqatning uyiga borish oldidan unga sovg'a olishmoqchi. Sovg'aning narxi `N` so'm. Shahbozda `A` so'm, Muhammadrizoda `B` so'm, Muhammadjonda `C` so'm pul bor. Ularning puli sovg'aga yetsa `Yes`, aks holda `No` degan yozuv chiqarilsin.
```
Comment (wahid-d) [inline Lab 5 - Shart Operatorlari.md:969]:
```suggestion
 > Shahboz, Muhammadrizo, Muhammadjon Furqatning uyiga borish oldidan unga sovg'a olishmoqchi. Sovg'aning narxi `N` so'm. Shahbozda `A` so'm, Muhammadrizoda `B` so'm, Muhammadjonda `C` so'm pul bor. Ularning puli sovg'aga yetsa `Yes`, aks holda `No` degan yozuv chiqarilsin.
```

File: Lab 5 - Shart Operatorlari.md [hunk 14]
```diff
@@ -891,3 +891,216 @@ int main()
...
+
+
```
Comment (wahid-d) [inline Lab 5 - Shart Operatorlari.md:971]: ortiqcha

File: Lab 5 - Shart Operatorlari.md [hunk 15]
```diff
@@ -891,3 +891,216 @@ int main()
...
+ ###### Input
+
+ Birinchi qatorda sovg'aning narxi `N`
+
+ Ikkinchi qatorda 3 ta butun son `A`, `B`, `C`.`(1 < A, B, C, N < 100000)`
```
Comment (wahid-d) [inline Lab 5 - Shart Operatorlari.md:976]:
```suggestion
 Ikkinchi qatorda 3 ta butun sonlar `A`, `B`, `C`: $1 \le A, B, C, N \le 100000$
```

File: Lab 5 - Shart Operatorlari.md [hunk 16]
```diff
@@ -891,3 +891,216 @@ int main()
...
+
+ ###### Output
+
+ `Yes` yoki `No` degan chop etilsin.
+
```
Comment (wahid-d) [inline Lab 5 - Shart Operatorlari.md:981]: ortiqcha

File: Lab 5 - Shart Operatorlari.md [hunk 17]
```diff
@@ -891,3 +891,216 @@ int main()
...
+ | Input | Output |
+ | - | - |
+ | `5`<br>`2 3 4` | `Yes` |
+
+ ###### Namuna 2
+
+ | Input | Output |
+ | - | - |
+ | `10`<br>`2 2 2`| `No` |
+
```
Comment (wahid-d) [inline Lab 5 - Shart Operatorlari.md:994]: ortiqcha

File: Lab 5 - Shart Operatorlari.md [hunk 18]
```diff
@@ -891,3 +891,216 @@ int main()
...
+
+ ---
+
+ ### Masala 25
+
+ ##### Ishora
```
Comment (wahid-d) [inline Lab 5 - Shart Operatorlari.md:1000]:
```suggestion
 ### Masala 25 - Ishora
```

File: Lab 5 - Shart Operatorlari.md [hunk 19]
```diff
@@ -891,3 +891,216 @@ int main()
...
+
+ ##### Description
+
+ >`2` ta berilgan sonning ishoralari bir xil bo'lsa `1`, ikkalasi `0` ga teng bo'lsa `0`, har xil bo'lsa `-1` chiqarilsin.
```
Comment (wahid-d) [inline Lab 5 - Shart Operatorlari.md:1004]:
```suggestion
 > `2` ta berilgan sonning ishoralari bir xil bo'lsa `1`, ikkalasi `0` ga teng bo'lsa `0`, har xil bo'lsa `-1` chiqarilsin.
```

File: Lab 5 - Shart Operatorlari.md [hunk 20]
```diff
@@ -891,3 +891,216 @@ int main()
...
+
+
+ ###### Input
+
+ Bitta qatorda `2` ta butun `a`, `b ` `(-100 < a, b < 100)`
```
Comment (wahid-d) [inline Lab 5 - Shart Operatorlari.md:1009]:
```suggestion
Bitta qatorda `2` ta butun `a`, `b `:  $-100 \le a, b \le 100$
```

File: Lab 5 - Shart Operatorlari.md [hunk 21]
```diff
@@ -891,3 +891,216 @@ int main()
...
+
+ ###### Output
+
+ Masalaning javobi
+
```
Comment (wahid-d) [inline Lab 5 - Shart Operatorlari.md:1014]: ortiqcha

File: Lab 5 - Shart Operatorlari.md [hunk 22]
```diff
@@ -891,3 +891,216 @@ int main()
...
+ ---
+
+
+### Masala 26
+
+##### Koordinata
+
+##### Description
+
+>Kordinatada bir nuqtani anglatuvchi `X` va `Y` ni o'qing. So'ngra shu nuqta kordinata tizimining qaysi choragiga tegishli ekanini aniqlab, chorak raqamini chop eting.
```
Comment (wahid-d) [inline Lab 5 - Shart Operatorlari.md:1037]:
```suggestion
> Kordinatada bir nuqtani anglatuvchi `X` va `Y` ni o'qing. So'ngra shu nuqta kordinata tizimining qaysi choragiga tegishli ekanini aniqlab, chorak raqamini chop eting.
```

File: Lab 5 - Shart Operatorlari.md [hunk 23]
```diff
@@ -891,3 +891,216 @@ int main()
...
+
+![Alt Text](https://oj.ilmhub.uz/public/upload/14810f34ef.png)
+
+
+`A`, `B`, `C`, `D` choraklar mos ravishda `1`, `2`, `3`, `4` ga teng.
+
+
+###### Input
+
+Bir qatorda ikkita butun  sonlar **A** va **B**.(`-10^1000 < A, B < 10^1000`)
```
Comment (wahid-d) [inline Lab 5 - Shart Operatorlari.md:1047]:
```suggestion
Bir qatorda ikkita butun  sonlar `A` va `B`: $-10^{1000} \le A, B \le 10^{1000}$
```

File: Lab 5 - Shart Operatorlari.md [hunk 24]
```diff
@@ -891,3 +891,216 @@ int main()
...
+
+| Nuqta kordinata o'qi ustida emas deb faraz qiling |
+| - | - |
+
+
+---
+
+### Masala 27
+
+##### Tenglik
```
Comment (wahid-d) [inline Lab 5 - Shart Operatorlari.md:1077]:
```suggestion
### Masala 27 - Tenglik
```

File: Lab 5 - Shart Operatorlari.md [hunk 25]
```diff
@@ -891,3 +891,216 @@ int main()
...
+
+##### Description
+
+>`4` ta berilgan sonlar bir-biriga teng bo'lsa `Yes` aks holda, `No` degan yozuv chiqaring
```
Comment (wahid-d) [inline Lab 5 - Shart Operatorlari.md:1081]:
```suggestion
> `4` ta berilgan sonlar bir-biriga teng bo'lsa `Yes` aks holda, `No` degan yozuv chiqaring
```

File: Lab 5 - Shart Operatorlari.md [hunk 26]
```diff
@@ -891,3 +891,216 @@ int main()
...
+
+
+###### Input
+
+Bir qatorda `4` ta butun son. (`1 <= a, b, c, d <= 10`)
```
Comment (wahid-d) [inline Lab 5 - Shart Operatorlari.md:1086]:
```suggestion
Bir qatorda `4` ta butun son: $1 \leq a, b, c, d \leq 10$
```

File: Lab 5 - Shart Operatorlari.md [hunk 27]
```diff
@@ -891,3 +891,216 @@ int main()
...
+| - | - |
+|`1 2 3 4`| `No` |
+
+###### Namuna 2
+
+| Input | Output |
+| - | - |
+|`1 1 1 1`| `Yes`|
+
+
```
Comment (wahid-d) [inline Lab 5 - Shart Operatorlari.md:1105]: orticha qatorlar

**Review by wahidustoz — CHANGES_REQUESTED** [review]
Change suggestions

**Review by wahidustoz — APPROVED** [review]

**Pull request merged**